app/assets/stylesheets/components/single_and_playlists_player/playlist.scss

```diff
@@ -55,4 +55,7 @@
       margin-right: auto;
     }
   }
+  .player .waveform {
+    padding-top: 14px;
+  }
```

Comment:
@ofsound I'm actually not sure why I had to do this, but it seemed necessary after switching to svg.